alpha/template/composite/builder.go

```diff
@@ -196,17 +205,28 @@ func (rb *RawBuilder) Build(dir string, td TemplateDefinition) error {
 		return fmt.Errorf("raw template configuration is invalid: %s", strings.Join(validationErrs, ","))
 	}
 
-	// build the container command
-	containerCmd := exec.Command(rb.builderCfg.ContainerCfg.ContainerTool,
-		"run",
-		"--rm",
-		"-v",
-		fmt.Sprintf("%s:%s:Z", rb.builderCfg.CurrentDirectory, rb.builderCfg.ContainerCfg.WorkingDir),
-		"--entrypoint=cat", // This assumes that the `cat` command is available in the container -- Should we also build a `... render-template raw` command to ensure consistent operation? Does OPM already have a way to render a raw FBC?
-		rb.builderCfg.ContainerCfg.BaseImage,
-		path.Join(rb.builderCfg.ContainerCfg.WorkingDir, rawConfig.Input))
+	istat, err := os.Stat(rawConfig.Input)
+	if err != nil {
+		return fmt.Errorf("raw template input file does not exist: %s, %v", rawConfig.Input, err)
+	}
+	if !istat.Mode().IsRegular() {
+		return fmt.Errorf("raw template input file is not a normal file: %s, %v", rawConfig.Input, err)
+	}
```

Comment:
It would completely fail, unless someone was super clever and provided a named pipe that they put in the dir that generated passable input schema... maybe this check isn't necessary.